About/style.css

```diff
@@ -0,0 +1,183 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@1,300&display=swap');
+@import url('http://fonts.cdnfonts.com/css/cocogoose?styles=44178');
+@import url('https://fonts.googleapis.com/css2?family=Lato:ital,wght@1,300&family=Recursive:wght@300&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  /* font-family: 'Lato', sans-serif; */
+  font-family: 'COCOGOOSE', sans-serif;
+}
+
+.background {
+  position: absolute;
+  top: 0;
+  width: 100%;
+}
+
+.logo {
+  width: 68px;
+  height: 33px;
+}
+
+.showcase {
+  background: #f6f6f7;
+  padding-top: 70px;
+  padding-bottom: 20px;
+}
+
+.description {
+  font-family: 'Recursive', sans-serif;
+  background: #fffefe;
+  border: 3px solid lightgrey;
+  padding: 10px 0;
+  font-size: 14px;
+}
+
+.p-span {
+  display: none;
+}
+
+.contact-info {
+  font-family: 'Recursive', sans-serif;
+  font-size: 14px;
+}
+
+.red {
+  color: #f2452c;
+  padding-bottom: 20px;
+}
+
+.global {
+  height: 1px;
+  width: 50px;
+  color: #f2452c;
+  font-weight: bold;
+  border-top: 3px solid rgb(242, 74, 54);
+  margin: auto;
+}
+
+.logo-history h1,
+.past h1 {
+  padding: 20px 0 5px 0;
+  font-size: larger;
+  font-family: 'Recursive', sans-serif;
+  font-weight: bold;
+}
+
+.logo-history img {
+  width: 320px;
+}
+
+.past {
+  padding-bottom: 30px;
+}
+
+.bg {
+  background-color: rgba(242, 74, 54, 0.7);
+}
+
+.card-border {
+  border: none;
+
+  /* margin: 50px 10px 0 50px;
+  width: 40%;
+  padding-left: 0;
+  height: 200px; */
+}
+
+/* .past img {
+  height: 200px;
+  object-fit: cover;
+} */
```

Comment:
- [ ] Kindly remove unnecessary comments from code that will make your code readable and clear